roles/telegraf/templates/fullnode/telegraf.conf.j2

```diff
@@ -188,3 +188,15 @@
 
   ## Destination bucket to write into.
   bucket = "supra-metrics" 
+
+[[outputs.influxdb_v2]]
+  urls = ["http://{{ monitor_server_dns }}:8086"]
+
+  ## Token for authentication.
+  token = "{{ vault.supra.internal.telegraf_token }}"
+  timeout = "15s"
+  ## Organization is the name of the organization you wish to write to; must exist.
+  organization = "{{ org }}"
+
+  ## Destination bucket to write into.
+  bucket = "supra-metrics"
```

Comment:
what about the labels that we usually have like, project and env ? can we add that while pushing to influxdb_v2